src/ostorlab/serve_app/oxo.py

```diff
@@ -372,6 +373,9 @@ def mutate(
             if asset.ip is not None:
                 new_asset = models.Network.create(networks=asset.ip)
                 created_assets.append(new_asset)
+            if asset.domain is not None:
+                new_asset = models.DomainAsset.create(domains=asset.domain)
```

Comment:
Why is this a list?

Reply:
Why not have a single DomainAsset and actually fix that also in network and return a list of Networks?

Reply:
Im not sure i do understand exactly your comment. 

it is a list because in the yaml file we provide it as a list, so we need to adapt  :
![image](https://github.com/Ostorlab/oxo/assets/144013278/6672162a-d232-41e1-8be4-33a6da053915)

and its the same for networks. If you check unit tests we do indeed return a list of networks/domains ..

If this doesnt answer your comment please clarify.

Reply:
Nothing says we should create ONE asset from that presentation.